tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/model/internal/UseDefaultLocationValidationService.java

```diff
@@ -63,7 +63,7 @@ protected Status compute()
         {
             if( ! NewLiferayPluginProjectOpMethods.canUseCustomLocation( op ) )
             {
-                retval = Status.createErrorStatus( "The sdk of lower version is not allowed to use custom location." );
+                retval = Status.createErrorStatus( "The sdk of specified version is not allowed to use custom location." );
```

Comment:
Msg should be "The specified SDK version is not allowed to use custom location."